samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an off-set.
+The off-set value may be due to the design of the radio circuit, the antenna used or the designed PCB Layout.
+Zero-distance value should be handled outside the library and sample.
+You should manually perform the calibration.
+To determine the off-set value, place the devices at reference distance and collect a batch of measurement data.
```

Comment:
Should we mention the suggested distance here, I think Carsten mentioned 60 cm?

Reply:
Good point. Added a note about the suggested distance.